clang/lib/DPCT/ASTTraversal.cpp

```diff
@@ -93,6 +93,7 @@ void MigrationRule::emplaceTransformation(TextModification *TM) {
 // RuleLang
 REGISTER_RULE(IterationSpaceBuiltinRule, PassKind::PK_Analysis)
 REGISTER_RULE(ErrorHandlingIfStmtRule, PassKind::PK_Migration)
+REGISTER_RULE(BinaryOperatorCallRule, PassKind::PK_Migration)
```

Comment:
Please give a right name for the rule,  w/o reading the code,  it is difficult to understand what the rule is doing.

Add necessary comments.

Reply:
Done.